src/main/kotlin/org/quiltmc/community/modes/quilt/extensions/github/GithubExtension.kt

```diff
@@ -0,0 +1,89 @@
+package org.quiltmc.community.modes.quilt.extensions.github
+
+import com.expediagroup.graphql.client.ktor.GraphQLKtorClient
+import com.kotlindiscord.kord.extensions.checks.hasRole
+import com.kotlindiscord.kord.extensions.commands.Arguments
+import com.kotlindiscord.kord.extensions.commands.application.slash.publicSubCommand
+import com.kotlindiscord.kord.extensions.commands.converters.impl.int
+import com.kotlindiscord.kord.extensions.commands.converters.impl.string
+import com.kotlindiscord.kord.extensions.extensions.Extension
+import com.kotlindiscord.kord.extensions.extensions.publicSlashCommand
+import com.kotlindiscord.kord.extensions.types.respond
+import io.ktor.client.*
+import io.ktor.client.engine.cio.*
+import io.ktor.client.features.*
+import io.ktor.client.request.*
+import org.quiltmc.community.GITHUB_TOKEN
+import org.quiltmc.community.TOOLCHAIN_GUILD
+import org.quiltmc.community.TOOLCHAIN_MODERATOR_ROLE
+import org.quiltmc.community.github.generated.DeleteIssue
+import org.quiltmc.community.github.generated.FindIssueId
+import java.net.URL
+
+class GithubExtension : Extension() {
+    private val client = GraphQLKtorClient(URL("https://api.github.com/graphql"),
+        HttpClient(engineFactory = CIO, block = {
+            defaultRequest {
+                header("Authorization", "bearer $GITHUB_TOKEN")
+            }
+        })
+    )
+    override val name = "github"
```

Comment:
This `val` should be at the top. Also, please put newlines between your declaration groups.